src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetVisitOrdersForPersonServiceTest.kt

```
@@ -109,4 +109,26 @@ class GetVisitOrdersForPersonServiceTest(
           type = UpstreamApiError.Type.BAD_REQUEST,
         ).shouldBe(true)
     }
+
+    it("return a 404 error when getVisitBalances data is null") {
+      whenever(nomisGateway.getVisitBalances(nomisNumber)).thenReturn(
+        Response(
+          data = null,
+          errors =
+            listOf(
+              UpstreamApiError(
```

Since we check for null and return the error in the check we don't need to return an error in this mock